avocado/core/nrunner.py

```diff
@@ -576,20 +575,47 @@ def unittest_name(self):
             if module.startswith(os.path.sep):
                 module = module[1:]
         module = module.replace(os.path.sep, ".")
-        return '%s.%s' % (module, class_method)
+        klass, method = class_method.rsplit('.', maxsplit=1)
```

Comment:
If i run:

```
avocado-runner-python-unittest runnable-run -u file.py:MyClass.SubClass.test_method
```

I get:

```
(Pdb) module
'file'
(Pdb) klass
'MyClass.SubClass'
(Pdb) method
'test_method'
```

Maybe check that both `klass` can not be nested here?

Reply:
Do we have examples in our documentations that would lead users to use this syntax?

IMO, trying to catch all possible unsupported syntaxes might create a complex code and will never cover all possible situations. So, I would suggest keep it simple and not check this.

Reply:
> Do we have examples in our documentations that would lead users to use this syntax?
> 

No, we don't and shouldn't.  

> IMO, trying to catch all possible unsupported syntaxes might create a complex code and will never cover all possible situations. So, I would suggest keep it simple and not check this.

That's fine.